skaffold.sh

```diff
@@ -1,27 +1,47 @@
 #!/usr/bin/env bash
-# Usage:
-# To disable metrics: development/minikube.sh
-# To enable metrics: METRICS=1 development/minikube.sh
 set -o errexit  # Used to exit upon error, avoiding cascading errors
 set -o pipefail # Unveils hidden failures
 set -o nounset  # Exposes unset variables
 
 PROFILE="${PROFILE:-kgw}"
 
-mkdir -pv /tmp/skaffold || echo '`/tmp/skaffold` already exist - skipping create'
+check_cluster_is_running() {
```

Comment:
This stuff should probably be in a different PR. It's causing noise here

Reply:
Yes, I agree.

It's not effecting anything though at the moment, so if it's OK, I'll just leave `skaffold.sh` as-is.